diff --git a/node_modules/pino-logflare/dist/httpStream.d.ts b/node_modules/pino-logflare/dist/httpStream.d.ts
index d726586..3a8b6df 100644
--- a/node_modules/pino-logflare/dist/httpStream.d.ts
+++ b/node_modules/pino-logflare/dist/httpStream.d.ts
@@ -1,6 +1,7 @@
+import { DestinationStream } from 'pino';
 import { LogflareUserOptionsI } from "logflare-transport-core";
 interface Options extends LogflareUserOptionsI {
     size?: number;
 }
-declare function createWriteStream(options: Options): any;
+declare function createWriteStream(options: Options): DestinationStream;
 export default createWriteStream;
diff --git a/node_modules/pino-logflare/dist/index.d.ts b/node_modules/pino-logflare/dist/index.d.ts
index 1886c06..5967f9d 100644
--- a/node_modules/pino-logflare/dist/index.d.ts
+++ b/node_modules/pino-logflare/dist/index.d.ts
@@ -2,9 +2,10 @@ import createHttpWriteStream from "./httpStream";
 import createConsoleWriteStream from "./consoleStream";
 import { Level, LogEvent } from "./utils";
 import { LogflareUserOptionsI } from "logflare-transport-core";
+import { DestinationStream } from 'pino';
 declare const createPinoBrowserSend: (options: LogflareUserOptionsI) => (level: Level, logEvent: LogEvent) => void;
 declare const logflarePinoVercel: (options: LogflareUserOptionsI) => {
-    stream: any;
+    stream: DestinationStream;
     send: (level: Level, logEvent: LogEvent) => void;
 };
 declare const createWriteStream: typeof createHttpWriteStream;
